djangocms_googlemap/__init__.py

```diff
@@ -1,3 +1,9 @@
-# -*- coding: utf-8 -*-
+"""
+Google Maps plugin for django CMS
+"""
 
+__author__ = 'Divio AG',
```

Comment:
Not sure I understand your comment. I know that this is different in other Divio packages.

(All this information makes sense in the package itself. It's included now in `setup.py`, just as `__version__`. Also note the docstring, which is also taken from the package and used as `description`  keyword in `setup()`.)

I see that I need to remove the trailing commas, though. I'll push a fix.